test/bench/etc/run_granular_benchmarks.js

```diff
@@ -28,6 +28,9 @@ const DOCUMENT_ROOT = path.resolve(`${__dirname}/../documents`);
     .run()
     .catch(() => null);
 
+  // Check for benchmark results
+  const cpuBaselineData = require(`${__dirname}${path.sep}cpuBaseline.json`);
```

Comment:
`require` is already script relative, so "./cpuBaseline.json" should be all that you need

But also, if you have the `path` module, might as well use `path.join()` there may be additional rules to "joining" that that methods handles that are more than putting the `.sep` in place.